src/Flex.php

```diff
@@ -198,7 +227,7 @@ public function executeAutoScripts(Event $event): void
         $this->io->write($this->postInstallOutput);
     }
 
-    private function filterPackageNames(PackageInterface $package, string $operation)
+    private function filterPackageNames(PackageInterface $package, string $operation): ?\Generator
```

Comment:
Fixed, but are you _absolutely certain_ that nothing in the `elseif` could have an edge case and therefore not return the expected `\Generator`?